medical_medicament/README.rst

```diff
@@ -6,42 +6,48 @@
 Medical Medicament
 ==================
 
-This module introduce the concept of Medicaments using product as basis
-It also includes concept of drug forms and drug routes
+This module introduce the concept of Medicaments using product as basis.
+It also includes concept of drug forms and drug routes.
 
 Usage
 =====
 
-For further information, please visit:
-
-* https://www.odoo.com/forum/help-1
-
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/159/8.0
+   :target: https://runbot.odoo-community.org/runbot/159/9.0
 
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
+.. branch is "9.0" for example
+
+Known issues / Roadmap
+======================
 
```

Comment:
Add a note that the storage column was removed from v8, and that the replacement is module `medical_medicament_storage`